core/transaction_pool.go

```diff
@@ -45,10 +46,21 @@ var (
 )
 
 const (
-	maxQueued = 64 // max limit of queued txs per address
+	maxQueued  = 64               // max limit of queued txs per address
+	resendTime = 39 * time.Second // every ~3 blocks
```

Comment:
If 3 blocks is desired this should be 45s since resendTime is compared against real time and it looks like we get 15s real time from the 13s target time.